src/main/java/mil/dds/anet/search/mssql/MssqlReportSearcher.java

```diff
@@ -85,7 +85,7 @@ public ReportList runSearch(ReportSearchQuery query, Handle dbHandle, Person use
 		searchBuilder.addDateClause(query.getEngagementDateStart(), Comparison.AFTER, "engagementDate", "startDate");
 		searchBuilder.addDateClause(query.getEngagementDateEnd(), Comparison.BEFORE, "engagementDate", "endDate");
 		searchBuilder.addDateClause(query.getCreatedAtStart(), Comparison.AFTER, "createdAt", "startCreatedAt");
-		searchBuilder.addDateClause(query.getCreatedAtStart(), Comparison.BEFORE	, "createdAt", "endCreatedAt");
+		searchBuilder.addDateClause(query.getCreatedAtEnd(), Comparison.BEFORE	, "createdAt", "endCreatedAt");
```

Comment:
Remove the spaces before the comma and I'm a happy man (as would eslint).

Reply:
done